.github/workflows/publish.js.yml

```diff
@@ -23,8 +23,6 @@ jobs:
         cache: gradle
     - run: npm install --no-package-lock
```

Comment:
shall we strip dev dependencies?

Reply:
It can by moving semantic release dependencies from devDependencies to dependencies